webui/src/locale/zh-CN.ts

```
@@ -7,6 +7,8 @@ import loginLocale from '@/views/login/locale/zh-CN'
 import oobeLocale from '@/views/oobe/locale/zh-CN'
 import topBanPageLocale from '@/views/ranks/locale/zh-CN'
 import ruleManageMentLocale from '@/views/rule-management/locale/zh-CN'
+import chartsLocale from '@/views/charts/locale/zh-CN'
```

Biome

[error] Shouldn't redeclare 'chartsLocale'. Consider to delete it or rename it.

'chartsLocale' is defined here:


(lint/suspicious/noRedeclare)